server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -3485,7 +3485,7 @@ private static void ensureNotAborted(ShardId shardId, SnapshotId snapshotId, Ind
                 // the ensureNotAborted() call above did not throw. The remaining options don't make sense, if they ever happen.
                 logger.error(
                     () -> Strings.format(
```

Comment:
This one is kinda nonsense? :) Why use a lambda and String.format here when you're not logging an exception?
I know it was like that before but maybe just remove the format call and lambda? :)

Reply:
Sure thing, 6a21cc98b3ff7403cce6a033b1d7a0f88f87c56d.